change/html/js/detailanswer.js

```
@@ -0,0 +1,135 @@
+                        .then(response => {
+                            if (!response.ok) {
+                                throw new Error('Network response was not ok');
+                            }
+                            return response.json();
+                        })
+                        .then(data => {
+                            const container = document.getElementById('questionsAndAnswers');
+                            container.innerHTML = ''; // 清空容器内容
+
+                            if (data && data.hasOwnProperty('id') && data.hasOwnProperty('title') && data.hasOwnProperty('answers')) {
+                                // 创建一个新的div来包裹问题和它的答案
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)